packages/sync-actions/test/product-types-sync-enums.spec.js

```diff
@@ -6,11 +6,8 @@ const createTestProductType = custom => ({
   ...custom,
 })
 
-const createAttributeDefinition = custom => ({
-  type: {
-    name: custom.name,
-  },
-  ...custom,
+const createAttributeDefinition = type => ({
```

Comment:
Do we need this function anymore? At this point it seems simpler to just inline the objects.